openapi/components/schemas/Invoices/TaxTypes/rebilly-avalara.yaml

```
@@ -0,0 +1,14 @@
+description: Avalara-based taxes.
+allOf:
+  - $ref: ../InvoiceTax.yaml
+  - type: object
+    properties:
+      items:
+        type: array
+        description: |-
+          Taxes that are applied to this invoice and calculated based
+          on the related product's `taxCategoryId`.
```

issue: [Possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes)


```suggestion
          on the related product `taxCategoryId`.
```